library/network_connections.py

```diff
@@ -467,6 +467,8 @@ def ifcfg_create(
                 ifcfg["DNS" + str(idx + 1)] = dns["address"]
             if ip["dns_search"]:
                 ifcfg["DOMAIN"] = " ".join(ip["dns_search"])
+            if ip["dns_options"]:
+                ifcfg["RES_OPTIONS"] = ""
```

Comment:
now this change is incorrect.

Anyway, `grep RES_OPTIONS -R /etc/sysconfig/` shows that initscripts don't support this option either. It's like with ipv6_disabled: the initscripts provider needs to reject specifying such settings.

Reply:
inside the VM, I get this:
```
[root@ibm-p8-kvm-03-guest-02 ~]# grep RES_OPTIONS -R /etc/sysconfig/ 
/etc/sysconfig/network-scripts/ifcfg-eth0-1:RES_OPTIONS="rotate timeout:1"
```